### ISSUE TITLE
defaultDate not working

### ISSUE THREAD

**Issue by RWOverdijk** [Eonasdan/bootstrap-datetimepicker#1997]
Awesome plugin :) One question though...

It looks like defaultDate is not working. I'm doing this:

```js
this.$datetimePicker = $(this.inputElement).datetimepicker({
  locale     : this.locale,
  defaultDate: new Date()
});
```

Everything works fine, except for defaultDate. If I remove it, I get the current date, which is nice, but I want to be able to set the defaultDate. Before I start digging in the code, any ideas?

**Comment by RWOverdijk** [comment]
FYI: It _does_ work when I use the method:

```js
this.$datetimePicker.data('DateTimePicker').defaultDate(this.value);
```

Does that help?

**Comment by RWOverdijk** [comment]
@William-H-M No if I use the defaultDate value, it doesn't set _anything_.

**Comment by RWOverdijk** [comment]
And I want to set the defaultValue (which is a date I got from the db, simple Date instance)

**Comment by William-H-M** [comment]
Hi, I understood now looks like I've got it, if it's something like that this can work
[Fiddle](http://jsfiddle.net/539mx1ja/2/)
It does work the same as with what you done throgh
`this.$datetimePicker.data('DateTimePicker').defaultDate(this.value);`

**Comment by RWOverdijk** [comment]
@William-H-M that's not the behavior I'm seeing. Is there perhaps something on master that hasn't been tagged yet?

**Comment by William-H-M** [comment]
@RWOverdijk Hi, the behavior that you're seeing is on the fiddle I gave or is on your copy of master?

**Comment by RWOverdijk** [comment]
In my application. I'm guessing I'm just doing something wrong... I'll keep using the .defaultDate() method instead.

**Comment by William-H-M** [comment]
Hi the problem it's probably that you have the old version of 4.17.44 that has this error and was fixed 4.17.45 and 4.17.46 so update your version of the plugin and if it's not that then keep using  .defaultDate() 
Have a nice day coding :)
